articles/azure/azs-how-link-storage-explorer.md

```diff
@@ -0,0 +1,60 @@
+---
+title: How to link Azure Storage Explorer to Azure Stack | UKCloud Ltd
+description: Provides help for linking Azure Storage Explorer to Azure Stack
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to link Azure Storage Explorer to Azure Stack
+toc_fullpath: Users/How To/Create a virtual machine/azs-how-link-storage-explorer.md
+toc_mdlink: azs-how-link-storage-explorer.md
+---
+
+# How to link Azure Storage Explorer to Azure Stack
+
+In this guide, you will learn how to link Azure Stack subscriptions to the Azure storage explorer. Azure storage explorer is a standalone application that allows you to easily work with Azure Stack storage data on Windows, macOS and Linux. This tool allows for easy downloading and uploading to Azure Stack which makes tasks such as downloading and uploading data disks to VM's very easy.
+
+If you need to install the storage explorer, please find the link [here](https://azure.microsoft.com/en-us/features/storage-explorer/) and install it.
+
+## Installation and setup
+
+1. Download and install storage explorer.
+
+    ![Storage tool installation guide 1](images/azs-browser-storage-tool-download.png)
+
+2. Once launched navigate to accounts.
+
+     ![Storage tool installation guide 2](images/azs-installation-storage-tool-1.png)
+
+3. Select **Add an account...**
```

Comment:
You don't need to include the ellipsis, so just Add an account. (Don't forget to put a full stop at the end of the sentence)

Reply:
On the storage explorer the button is called ""Add an account...", If you want the ellipsis removed Im happy to do so, I was just keeping it in line with what the user will see

Reply:
It's standard practice to not include ellipses from field labels/buttons/links, etc. in documentation.